CHANGELOG.md

```diff
@@ -27,7 +27,7 @@
 ### BREAKING CHANGES
 
 * **react-chart:**
-Previously the `ArgumenAxis`, `ValueAxis`, and `Scale` plugins allowed you to customize argument and value scales. In this release, we have implemented the `ArgumentScale` and `ValueScale` plugins for this purpose to make the API more clear. 
+Previously the `ArgumentAxis`, `ValueAxis`, and `Scale` plugins allowed you to customize argument and value scales. In this release, we have implemented the `ArgumentScale` and `ValueScale` plugins for this purpose to make the API more clear.
```

Comment:
"more clear" First of all, you should use 'clearer'. However, this does not actually explain anything.